bitcoin/_posts/0000-00-00-advent-27.md

```diff
@@ -0,0 +1,287 @@
+**Rebuttal:**\
+_If the issue is a bug, it would merit more strict scrutiny on the code and
+tests (which are pretty comprehensive currently) as to how they could be passing
+with an issue like that. Once patched and reviewed, it's still possible to merge
+and release. However, the issue could also be a conceptual bug (e.g., here's how
+to do recrusive covenants with CTV), which would certainly delay (and I'd be the
+first to say it!) continuing with CTV at the present time. The former issue is
+likely more likely than the latter, and that risk is defrayed by thorough code
+review and testing._
+1. **Concern:** There's not developer consensus conceptually.\
+**Rebuttal:**\
+_There are many developers supporting on
+[utxos.org/signals](https://utxos.org/signals). I've also made an effort to
+reach out to a variety of developers who are not on the site to seek out
+dissent, and I do not think there are any current safety concerns, which is
+positive for developers who might not Ack but don't have a 'this is a grave
+mistake' Nack up their sleeves._
+1. **Concern:** Miners don't want CTV and don't signal for it.\
+**Rebuttal:**\
+_20% of the hash pools already say they do want it, and more should be joining
+that list soon™._
+1. **Concern:** Disagreement on Activation mechanism.\
+**Rebuttal:**\
+_Taproot's Speedy Trial worked very well. Big Success. No need to mess with the
+recipe. UASF-clienters know how to make the mods to the activation logic for
+whatever they want as a competing client. But perhaps that's not a comfortable
+status quo, so this is certainly one to watch._
+
+## And Then What?
+
+After the release of CTV Soft Fork Client, what goes next?
+
+1. Jeremy takes a vacation for a month.
+1. Sapio continues to improve dramatically, as detailed in the last post.
+1. Work begins on new Opcodes.
+1. More fun applications get built on CTV/Sapio.
+1. Non-Interactive Lightning Channel BOLT drops.
+1. Payment Pool spec drops with LN integration.
+1. (Privacy + Scale)++
+
+## New Opcodes you say?
+
+Yep, new opcodes. As soon as CTV is merged, there are some new features that
+could be tooled into BIPs without much controversy, given their simplicity:
+
+1. New Math Opcodes
+1. OP_AMOUNT
+1. OP_CSFS
+
+These are "universally agreed on" bits of functionality that have very little
+wiggle room in how they might be specified. Therefore their implementation,
+testing, and release is mostly mechanical. This makes them good bets for a
+concrete timeline because they're a rote-development task with few research
+dependencies and easily agreed on design.
+
+With hard work, these could be reviewed and tested in time for Speedy Trial
+signalling in June 2023 (+1 year), with realllllly hard work 6 months earlier
+(but there are conflicting priorities for developer's time -- e.g., building
+stuff with/for CTV and Taproot -- that make that super unlikely).
+
+### What about Anyprevout
+
```

Comment:
If this hasn't already shipped, one point you may want to attempt to get across here is that any sort of "next-ism" is obviated by the fact that it's possible to attempt parallel soft-forks with distinct version bits. The possibility of parallel soft forks means things can be ultimately "proposed" whenever they're mature enough to enter the "final stage".

Reply:
great point.